src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotate.java

```diff
@@ -0,0 +1,479 @@
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.UTR, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "Gene(s) for which the SV is predicted to disrupt a UTR."));
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.MSV_EXON_OVERLAP, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "Gene(s) on which the multiallelic SV would be predicted to have a LOF, INTRAGENIC_EXON_DUP, COPY_GAIN, DUP_PARTIAL, or PARTIAL_EXON_DUP annotation if the SV were biallelic."));
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.PROMOTER, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "Gene(s) for which the SV is predicted to overlap the promoter region."));
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.BREAKEND_EXON, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "Gene(s) for which the SV breakend is predicted to fall in an exon."));
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.INTERGENIC, 0, VCFHeaderLineType.Flag, "SV does not overlap coding sequence."));
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.NONCODING_SPAN, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "Class(es) of noncoding elements spanned by SV."));
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.NONCODING_BREAKPOINT, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "Class(es) of noncoding elements disrupted by SV breakpoint."));
+        header.addMetaDataLine(new VCFInfoHeaderLine(GATKSVVCFConstants.NEAREST_TSS, VCFHeaderLineCount.UNBOUNDED, VCFHeaderLineType.String, "Nearest transcription start site to intragenic variants."));
+
+    }
+
+    private void updateAndWriteHeader(VCFHeader header) {
+        addAnnotationInfoKeysToHeader(header);
+        vcfWriter.writeHeader(header);
+    }
+
+    protected static boolean variantSpansFeature(final SimpleInterval variantInterval, final SimpleInterval featureInterval) {
+        return variantInterval.contains(featureInterval);
+    }
+
+    protected static int countBreakendsInsideFeature(final SimpleInterval variantInterval, final SimpleInterval featureInterval) {
+        int count = 0;
+        if (variantInterval.getContig().equals(featureInterval.getContig())) {
+            if (variantInterval.getStart() >= featureInterval.getStart() && variantInterval.getStart() <= featureInterval.getEnd()) {
+                count++;
+            }
+            if (variantInterval.getEnd() >= featureInterval.getStart() && variantInterval.getEnd() <= featureInterval.getEnd()) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    protected static boolean variantOverlapsFeature(final SimpleInterval variantInterval, final SimpleInterval featureInterval) {
```

Comment:
Pretty sure this method is unnecessary (and is, in fact, causing you to create an extra SimpleInterval).  I think you can just call variantInterval.overlaps(someFeature), rather than variantOverlapsFeature(new SimpleInterval(someFeature)).